tests/integration_tests/cpu/propagator/jacobian_validation.cpp

```diff
@@ -1645,18 +1670,20 @@ int main(int argc, char** argv) {
         tel_det_config<rect_type, detail::helix<transform3_type>> rectangle_cfg{
             rect, helix_bz};
         rectangle_cfg.envelope(envelope_size);
+        rectangle_cfg.module_material(vacuum<scalar>{});
+        rectangle_cfg.mat_thickness(0.f);
         rectangle_cfg.n_surfaces(2u)
             .length(detector_length)
             .volume_material(vacuum<scalar>{});
```

Comment:
Volume material is set to vacuum here. Does this not mean that there is in fact no material in the detector and the check was correct to produce a warning?